src/store/reducers/walletConnectionsCache/walletConnectionsCache.types.ts

```diff
@@ -1,16 +1,17 @@
-// TODO: mock data type for connect wallet ui. Need update after core function completed.
 interface ConnectionData {
-  id: number | string;
-  name: string;
-  owner: string;
-  image?: string;
-  url: string;
+  id: string;
+  name?: string;
+  url?: string;
+  createdAt?: Date;
+  iconB64?: string;
+  selectedAid?: string;
```

Comment:
I think we select the AID before connecting, right? So this shouldn't be optional

Reply:
Yeah, we select the AID before connecting but we only update the selected Aid after it connects successfully.